.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+
+node_js:
+  - 4
+  - 5
+  - 6
```

Comment:
Don't you need to specify how to run the tests themselves? Like `node test` perhaps?